### ISSUE TITLE
Map `bounds` prop not updating in version 1.0 

### ISSUE THREAD

**Issue by prl900** [emilhe/dash-leaflet#194]
Not sure if this is due to a change in the behavior of Leaflet but the `bounds` property doesn't seem to be set in the new version of dash-leaflet.

Before, the `bounds` property was firing every time the map area changed due to zooming or panning.

Here is a sample code to reproduce the problem:
```
from dash import Dash, callback, html, Input, Output
import dash_leaflet as dl

app = Dash()
app.layout = html.Div([
    dl.Map(dl.TileLayer(), id='map', center=[56,10], zoom=6, style={'height': '50vh'}),
    html.Div(id="text")
])

@callback(
    Output("text", "children"),
    Input("map", "bounds"),
)
def update(bounds):
    print("AAA", bounds)
    return bounds

if __name__ == '__main__':
    app.run_server()
```

When panning/zooming the callback doesn't fire. The initial callback reports `bounds=None` .

Any help with this to solve or identify an alternative prop to track the map's bounds would be greatly appreciated.

**Comment by emilhe** [comment]
Thanks for the feedback and a great MWE! You are correct - viewport tracking wasn't implemented as part of the `1.0.0` release. I have now added code that should yield similar behavior as before, i.e. updating the `zoom`, `center`, and `bounds` properties of the map. If you install the latest version,

```
pip install dash-leaflet==1.0.5
```

and serialize the callback output,

```
import json
...
return json.dumps(bounds)
...
```

your example should work. Note that the `viewport` property was removed in recent versions of React Leaflet, which is why it is not longer available in `dash-leaflet` after the 1..0.0 release.
